test/evergreen.yml

```diff
@@ -1783,6 +1792,31 @@ tasks:
             set -o verbose
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
+  
+  - name: format-stress-sanitizer-test
+    commands:
+      - func: "get project"
```

Comment:
Can you break the get and compile stuff into a different task ?

Reply:
Right now there is no other task using this particular compilation combination. Maybe if there are more than one task in future that use this compilation step then we can make a separate task and make this task depend on that.